commands/hugo.go

```diff
@@ -781,6 +781,15 @@ func (c *commandeer) newWatcher(dirList ...string) (*watcher.Batcher, error) {
 						if ev.Op&fsnotify.Chmod == fsnotify.Chmod {
 							continue
 						}
+						if ev.Op&fsnotify.Remove == fsnotify.Remove {
+							for _, configFile := range c.configFiles {
+								c.Logger.FEEDBACK.Println(configFile, "disappeared, waiting for it to come back...")
+								for watcher.Add(configFile) != nil {
+									time.Sleep(100 * time.Millisecond)
+								}
+								c.Logger.FEEDBACK.Println(configFile, "is back!")
```

Comment:
I'm not a fan of these funny log messages, esp since you're not really "waiting for it to come back" (which is fine)

I dont' see the full code in this diff, but this event is the config file only, right? We're not adding a sleep fore every remove event?

Reply:
@bep Thanks for the review!

> I'm not a fan of these funny log messages, esp since you're not really "waiting for it to come back" (which is fine)

Haha, true!  The log messages are not really needed, but just for my debugging.
I'll push this commit again with the funny log messages removed.

> I dont' see the full code in this diff, but this event is the config file only, right? We're not adding a sleep fore every remove event?

Yes, this event is for the config file only, being within the existing if-block for config files:

```go
    if configSet[ev.Name] {
        // if Chmod event, skip (continue)
        // if Remove event, sleep and add it back
        // Config file changed. Need full rebuild.
    }
```